src/test/java/com/kodilla/ecommercee/GroupCrudTestSuite.java

```diff
@@ -81,24 +79,7 @@ public void testGroupDeleteOne() {
         genericEntityRepository.delete(groupFood);
```

Comment:
@jalowiec Przepraszam, że od razu tego nie zauważyłam :( Jeszcze brakuje usunięcia groupClothes, żeby test nie pozostawiał po sobie śladu.

Reply:
Jeśli dodana jest adnotacja `@Transactiol` to usuwanie nie jest konieczne - automatycznie zostaną usunięte zapisane dane